cypress/support/commands.js

```diff
@@ -762,6 +762,12 @@ function createHtmlPasteEvent(htmlContent) {
   );
 }
 
+Cypress.Commands.add('addNewBlock', (blockName, createNewSlate = false) => {
+  let block;
+  block = cy.getSlate(createNewSlate).type(`/${blockName}{enter}`);
+  return block;
+});
```

Comment:
I like the addition of this command, but it means most of the tests are now implicitly testing the slash menu instead of the block chooser. Do we need to add an end-to-end test for the block chooser specifically, or is there still one that covers it?

Reply:
Yes, we need to test the blockchooser in addition, although I'd generalise the new command for normal block creation.

Reply:
I agree on that, it's a good idea.